src/tracer.js

```diff
@@ -272,7 +271,14 @@ export default class Tracer {
             throw new Error(`Unsupported format: ${format}`);
         }
 
-        spanContext.samplingFinalized = true;
+        // Here because opentracing api compatibility tests can pass
+        // either span or span context.  Calling finalizeSampling below
+        // only works on a span context.
```

Comment:
irrelevant comment. OT API allows span instance to be passed into inject(), so this takes care of that. Has nothing to do with compatibility tests or finalizeSampling